ads/_ping_.js

```diff
@@ -18,10 +18,13 @@ import {validateData} from '../3p/3p';
 import {dev} from '../src/log';
 
 /**
+ * A fake ad network integration that is mainly used for testing
+ * and demo purpose.
```

Comment:
nit: "demo purposes" in plural

Also add "This implementation gets stripped out in compiled production code."